src/data.yml

```diff
@@ -152,11 +152,11 @@ mailProviders:
       (Entry Plan, €2.50/m)
     level: 2
   aliases:
-    text: Yes (Entry Plan, €2.50)
+    text: Yes (Entry Plan, €2.50/m)
     level: 2
   webClientAccess:
-    text: Yes (IMAP, POP3, SMTP)
-    level: 1
+    text: Yes (Entry Plan, €2.50/m)

```

Comment:
I think this should also still include the supported protocols, to stay consistent

Reply:
Dw, since you deleted your branch, I'll merge this anyway, and edit later